### DISCUSSION TITLE
Examples link 404s in the Getting Started menu

### DISCUSSION THREAD

**Discussion by ashfyre** [FirstGearGames/FishNet#643] [Documentation]
I'm unsure whether or not to post this in the main fishnet repo or in the documentation repo, but it looks like the Fish-Networking -> Getting Started -> Examples link 404s. It is supposed to lead to [https://fish-networking.gitbook.io/docs/manual/tutorials/example-projects](https://fish-networking.gitbook.io/docs/manual/tutorials/example-projects) - unfortunately, that doesn't seem to exist anymore. Is there another place it should be linking to? Or is there no longer any example content?

**Comment by FirstGearGames** [comment]
They do not exist anymore. Demos are in the FN folder and examples were removed. Thanks for finding this, it's been fixed in recent releases.
